context/acceptencoder.go

```diff
@@ -25,8 +25,36 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+
+	"github.com/astaxie/beego/config"
+)
+
+var (
+	//Content will only be compressed if content length is either unknown or greater than gzipMinLength.
+	gzipMinLength int
+	//Default size==20B same as nginx
+	defaultGzipMinLength = 20
+	//The compression level used for deflate compression. (0-9).
+	gzipCompressLevel int
+	//List of HTTP methods to compress. If not set, only GET requests are compressed.
+	includedMethods map[string]bool
+	getMethodOnly   bool
 )
 
+func InitGzip(cf config.Configer) {
```

Comment:
it lead into the config package into context. actually I want to keep all modules independence. is it possible we just pass the value while not config.Configer.